src/gui/mainwindow.cpp

```diff
@@ -1469,6 +1469,7 @@ void MainWindow::activate()
 
 void MainWindow::optionsSaved()
 {
+    Logger::instance()->addMessage(tr("Options saved."));
```

Comment:
Shouldn't we use "Settings" term?

Reply:
I don't think so, it corresponds to the Options dialog. And I suppose users are already accustomed with the "Options" term.